usecases/MinimumWorkingExample/upload_scripts/doit_upload.py

```diff
@@ -517,7 +680,84 @@ def _emodul_upload(
     return emodul_sample
 
 
-def _actions_run(
+def _ingredient_upload(
```

Comment:
In the webview of the mat-o-lab openbis instance I don't see the annotation lables. Why? Did you upload the data again?